CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.4]
+
+- Implement PrintSchemaAndCapabilities command
```

Comment:
Can this be more descriptive so that end users understand what this means better?

Reply:
Improved